split up ecsql reference into indiviual files, remove iframed iModel … (#6377)
iTwin/itwinjs-core: docs/changehistory/4.3.0.md

ECSQL supports querying instance properties, which are any property in a class selected in ECSql or its derived classes.

[**ECSQL Instance Properties Documentation**](../learning/ECSqlReference/InstanceQuery.md)

### Window functions

> ECSQL now supports [window functions](../learning/ECSqlReference/WindowFunctions.md). **This functionality add following new keywords that might collide with alias, parameter, property or class name and may break existing queries.** Application need to [escape](../learning/ECSqlReference/ECSqlKeywords.md#escaping-keywords) those name or try escaping all names in their queries to be on safe side in future.

Reviewer: ```suggestion
ECSQL now supports [window functions](../learning/ECSqlReference/WindowFunctions.md). **This functionality adds the following new keywords that might collide with alias, parameter, property or class names and may break existing queries.** Applications should [escape](../learning/ECSqlReference/ECSqlKeywords.md#escaping-keywords) those names (or try escaping all names in their queries) to be on the safe side in the future.
```
